zebra/rt_netlink.c

```diff
@@ -2274,6 +2274,25 @@ static int netlink_macfdb_update(struct interface *ifp, vlanid_t vid,
 			    0);
 }
 
+static void netlink_handle_5549(struct ndmsg *ndm, struct zebra_if *zif,
+				struct interface *ifp, struct ipaddr *ip)
+{
+	char b[16] = "169.254.0.1";
+	struct in_addr ipv4_ll;
```

Comment:
just for my understanding, this hard coded value is related to rfc5549 ?
I already saw that hardcoding somewhere else in zebra. perhaps does it make sense to later have a common define?

Reply:
Yes, related to the actions required to do the rfc5549 prefix installation due to kernel limitations.  I can make this a #define if necessary but hopefully this will disappear completely with the new kernel support for actual ipv4 prefixes with ipv6 nexthops.  Donald probably knows when that support is expected to be available.

Reply:
Chatted with Donald and he stated he preferred the current approach of defining the char buffer.  Is this a strong requirement from you for the commit?

Reply:
Just one nit: I think you can reuse the global `ipv4_ll` variable instead of defining a new one here. Other than that this commit looks good to me.

Reply:
Thanks.  Missed the fact that the global exists.  Will do

Reply:
Comments added, Philippe.  Need anything else?